lib/tasks/certs.rake

```diff
@@ -207,26 +207,44 @@ namespace :certs do
     end
   end
 
-
-
-  desc 'Check that LG certificate bundle matches certificates in certificate path'
+  desc 'Validate certificate bundle as matching certificate path and inclusion in FICAM'
```

Comment:
I forget -- how close are we to having those `cert_bundles` generated as part of the deploy process?

Reply:
I think we'll sooner move to using a different bundle ([related Slack discussion](https://gsa-tts.slack.com/archives/C05MGJ72GU9/p1717434742258289?thread_ts=1717433534.870859&cid=C05MGJ72GU9)), which will probably still require some maintaining that we could/should incorporate into the deployment process, but wouldn't want to plan too much around it since it might work a little differently.

I'll create a ticket for it so that we don't lose track.